backend/fms_core/models/sample_property.py

```diff
@@ -0,0 +1,19 @@
+import reversion
+
+from .tracked_model import TrackedModel
+from django.db import models
+from .sample import Sample
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._validators import name_validator
+
+
+@reversion.register()
+class SampleProperty(TrackedModel):
+    """ Class to store additional sample metadata. """
+
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, validators=[name_validator],
```

Comment:
i would add a unique constraint with the sample id in order to have just one of a given metadata name by sample ...
```
class Meta:
    constraints = [
            models.UniqueConstraint(fields=['name', 'biosample_id'], name='unique_metadata_name_by_biosample_id')
      ]

```